### PR TITLE
DEVX-1777: CWD relative script access prevents some scripts running w…

### PR DESCRIPTION
**Pull request by ybyzek**
…hen called from arbitrary dir

### Description 

https://confluentinc.atlassian.net/browse/DEVX-1777

_What behavior does this PR change, and why?_

Minor cleanup

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Run cp-demo

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

File: scripts/validate/validate_bindings.sh [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
```
Comment (mikebin) [inline scripts/validate/validate_bindings.sh:3]: Is this needed anymore?

**Review by mikebin — APPROVED** [review]
LGTM, just one minor question.

**Review by javabrett — APPROVED** [review]

File: scripts/validate/validate_bindings.sh (hunk 1 above)
Comment (ybyzek) [reply scripts/validate/validate_bindings.sh:3]: @mikebin as the code is today, `DIR` is used in the next line.

Comment (mikebin) [reply scripts/validate/validate_bindings.sh:3]: Oops, missed that. Sorry for the noise.

**Pull request merged**